packages/components/src/draggable/index.js

```diff
@@ -15,7 +18,9 @@ const cloneHeightTransformationBreakpoint = 700;
 const clonePadding = 20;
 
 const isChromeUA = ( ) => /Chrome/i.test( window.navigator.userAgent );
-const documentHasIframes = ( ) => [ ...document.getElementById( 'editor' ).querySelectorAll( 'iframe' ) ].length > 0;
+const documentHasIframes = () => {
+	return !! invoke( document, [ 'body', 'querySelector' ], 'iframe' );
+};
```

Comment:
@nosolosw Any reason this check was editor specific? Is it because WP can use iframe outside of the "content" canvas (maybe in menus, topbars)? What impact could this have?

Reply:
Exactly. My old self did document this as problematic: https://github.com/WordPress/gutenberg/pull/9511#discussion_r214390996

If we do this, what will happen in chrome browsers that don't have the fix yet ([<72 if I'm reading correctly the upstream bugfix](https://bugs.chromium.org/p/chromium/issues/detail?id=737691#c8)?) is that it'll fire the `onDrop` and then `onDragEnd`, effectively executing the `onDragEnd` property twice.

An alternative fix would be to keep track of whether the drag operation was already reseted keeping a `isDragging` variable similar to `isChromeAndHasIframes`.

Reply:
@nosolosw, @youknowriad  Can we just remove this logic and assume the problem is fixed in chrome? I guess the last two versions that we need to support already have this problem fixed given that the problem was fixed in Wed, Nov 21, 2018, and chrome updates are fairly regular.

Reply:
@jorgefilipecosta That makes sense to me

Reply:
Agreed to that: https://github.com/WordPress/gutenberg/pull/15665 (wanted to test this effectively had been fixed so I had to prepare the revert anyway).